chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/activity/ActivityConcept.java

```diff
@@ -4,24 +4,11 @@
 
 /**
  * Different types of activity the system will log.
+ * 
  * @author kekey
  *
  */
 public enum ActivityConcept implements Serializable {
 
-    CERTIFIED_PRODUCT,
-    PRODUCT,
-    DEVELOPER,
-    CERTIFICATION,
-    CQM,
-    CERTIFICATION_BODY,
-    VERSION,
-    USER,
-    TESTING_LAB,
-    PENDING_CERTIFIED_PRODUCT,
-    API_KEY,
-    ANNOUNCEMENT,
-    CERTIFICATION_ID,
-    PENDING_SURVEILLANCE,
-    CORRECTIVE_ACTION_PLAN;
+	CERTIFIED_PRODUCT, PRODUCT, DEVELOPER, CERTIFICATION, CQM, CERTIFICATION_BODY, VERSION, USER, TESTING_LAB, PENDING_CERTIFIED_PRODUCT, API_KEY, ANNOUNCEMENT, CERTIFICATION_ID, PENDING_SURVEILLANCE, CORRECTIVE_ACTION_PLAN, COMPLAINT;
```

Comment:
Did this need to be changed? I feel like one per line is SO much easier to read.